modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+                        }'
+----
+
+*Example:* Conflict logging is enabled, and the default conflict collection is `conflictlogsbucket.myappscope.log1collection`. 
+All conflicts in the replication will be logged to the default conflict collection.
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                        "disabled": false,
+                        "bucket": "conflictlogsbucket",
+                        "collection": "myappscope.log1collection"
+                        }'
+----
+
+[#enabled-with-special-logging-rules]
+==== Enabled with special logging rules
+
+To specify different conflict collection rules for different scopes and collections in your replication, define `loggingRules` in the `conflictLogging` settings. 
+
+You can use `loggingRules` to specify, for any scope or collection in the replication, whether the conflicts from that scope or collection must be logged to a different conflict collection or not.  
+If you do not specify in the logging rules, then by default, the conflicts from all collections are logged to the default conflict collection.
+
+*Format:*
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                    "disabled": false,
+                    "bucket": [bucketName],
+                    "collection": [scope].[collection],
+                    "loggingRules": {[logging_rules]},
+                    }'
+----
+
+*Example:* Conflict logging is enabled. The default conflict collection is the same as in the previous example, however, `loggingRules` is added. 
+The `conflictLogging` setting defines the following rules:
+
+* Conflict logs for the collection `agents.partners` must be logged to a different collection called `conflictlogsbucket2.myappscope.partnerlogs`
+
+* Conflicts must not be logged in the collection `agents.retailers`.
+
+* Conflict logs for the `inventory` scope must be logged to a collection called `conflictlogsbucket2.myappscope.inventorylogs`.
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+"disabled": false,
+"bucket": "conflictlogsbucket1",
+"collection": "myappscope.log1collection"
+"loggingRules": {
+      "agents.partners": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.partnerlogs"
+      },
+      "agents.retailers": null,
+      "inventory": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.inventorylogs"
+      }
+    }
+}'
+----
+
+[#disabled-with-stored-logging-rules]
+==== Disabled with stored logging rules
+
+You can configure `conflictLogging` (specify the default conflict collection and the `loggingRules`) but not enable `conflictLogging` by setting `disabled` to `true`. 
+
+NOTE: You can update the replication and set the `conflictLogging` status `disabled` to `true` or `false` at any time.
+
+When you set `"disabled": true`, it also disables conflict logging, but all of the rules are retained for the collection.
```

Comment:
When you set `"disabled": true`, but continue to specify the other fields, the specified configurations will be retained.